test/unit/context-map/test-workload.js

```diff
@@ -46,10 +53,12 @@ describe('context-map/workload', function () {
       workloadWith24Cases.casesSummary.communityTiers = stagingHelper.getCountableTestTiers(Locations.COMMUNITY)
       workloadWith24Cases.casesSummary.custodyTiers = stagingHelper.getCountableTestTiers(Locations.CUSTODY)
       workloadWith24Cases.casesSummary.licenseTiers = stagingHelper.getCountableTestTiers(Locations.LICENSE)
+      workloadWith24Cases.casesSummary.t2aCommunityTiers = stagingHelper.getCountableTestTiers(Locations.COMMUNITY)
+      workloadWith24Cases.casesSummary.t2aCustodyTiers = stagingHelper.getCountableTestTiers(Locations.CUSTODY)
+      workloadWith24Cases.casesSummary.t2aLicenseTiers = stagingHelper.getCountableTestTiers(Locations.LICENSE)
 
       var mappedWorkloadWith24Cases = mapper(workloadWith24Cases, ownerId, workloadReportId)
-
-      expect(mappedWorkloadWith24Cases.totalCases).to.equal(24)
+      expect(mappedWorkloadWith24Cases.totalCases).to.equal(48)
```

Comment:
Minor: This workload now has 48 cases so `mappedWorkloadWith24Cases` should be updated.

Reply:
good catch!